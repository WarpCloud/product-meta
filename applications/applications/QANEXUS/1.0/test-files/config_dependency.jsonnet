{
  application_type: "QANEXUS",
  application_name: "nexus",
  application_version: "1.0",
  user_config: {},
}
