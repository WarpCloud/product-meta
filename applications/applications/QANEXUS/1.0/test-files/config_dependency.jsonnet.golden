{
   "Customized_Namespace": "default",
   "Develop": false,
   "Transwarp_Install_ID": "",
   "instance_list": [
      {
         "annotations": {
            "application_name": "nexus",
            "application_type": "QANEXUS"
         },
         "configs": {
            "nexus_cpu_limit": 2,
            "nexus_cpu_request": 0.10000000000000001,
            "nexus_memory_limit": 4,
            "nexus_memory_request": 1
         },
         "instance_settings": { },
         "moduleName": "qa-nexus",
         "name": "nexus-qa-nexus",
         "version": "1.0"
      }
   ],
   "instance_settings": { },
   "version": ""
}
